### PR TITLE
fix indention of values which are object and have multiple member

### LINKED ISSUE #25: Support pretty for struct in struct
## Currently

```swift
let owner = Owner(name: "Nanachi", age: 20, address: "4th layer in Abyss")
let dog = Dog(name: "Pochi", owner: owner, age: nil)
```

```swift
elementString(dog, debug: false, pretty: true)
// =>
// Dog(name: "Pochi",
//     owner: Owner(name: "Nanachi",
//           age: 20,                         // invalid indent
//           address: "4th layer in Abyss"),  // 
//     age: nil)
```

## Expected

```swift
elementString(dog, debug: false, pretty: true)
// =>
// Dog(name: "Pochi",
//     owner: Owner(name: "Nanachi",
//                  age: 20,
//                  address: "4th layer in Abyss"),
//     age: nil)
```

## Reference
https://github.com/YusukeHosonuma/SwiftPrettyPrint/pull/26/files#diff-9ab6ad81e2adbf8e1be1f48cfc11cf1dR42-R52

### PR DESCRIPTION
**Pull request by sahara-ooga**
As title, I fixed indention for `objectString()` when the member is object and it has multiple members.

resolve #25

### CODE REVIEW

**Comment by sahara-ooga** [comment]
After merge of master brunch, a test case starts to fail.
So I close this PR temporarily.

**Comment by sahara-ooga** [comment]
Fix bug (b41b3c9) and pass tests.

File: Sources/Core/Formatter/MultilineFormatter.swift [hunk 1]
```diff
@@ -39,10 +39,31 @@ class MultilineFormatter: PrettyFormatter {
...
+    /// goes to
+    ///
+    /// owner: Owner(name: "Nanachi",
+    ///              age: 4))
+    ///
+    /// - Parameter source: key value pair
+    private func indentInsertedKeyValueString(_ source: (String, String)) -> String {
+        let indentInsertedNewLineCharactor = "\n" + String(repeating: " ", count: "\(source.0): ".count)
+        let indentInserted = source.1.replacingOccurrences(of: "\n", with: indentInsertedNewLineCharactor)
+        return "\(source.0): \(indentInserted)"
```
Comment (YusukeHosonuma) [inline Sources/Core/Formatter/MultilineFormatter.swift]: Can you use `String#indentTail` instead to more simply?

e.g.

```swift
source.1.indentTail(size: "\(source.0): ".count) // I'm not confirm to this :)
```

File: Sources/Core/Formatter/MultilineFormatter.swift [hunk 2]
```diff
@@ -39,10 +39,31 @@ class MultilineFormatter: PrettyFormatter {
...
+    ///       age: 4)
+    /// """)
+    ///
+    /// goes to
+    ///
+    /// owner: Owner(name: "Nanachi",
+    ///              age: 4))
+    ///
+    /// - Parameter source: key value pair
+    private func indentInsertedKeyValueString(_ source: (String, String)) -> String {
```
Comment (YusukeHosonuma) [inline Sources/Core/Formatter/MultilineFormatter.swift]: Specify arguments name to tuple to more readable ? if possible :)

e.g. `_ source: (label: String, value: String)`

**Review by YusukeHosonuma — COMMENTED** [review]
Thanks for bugfix!

File: Sources/Core/Formatter/MultilineFormatter.swift (hunk 1 above)
Comment (sahara-ooga) [reply Sources/Core/Formatter/MultilineFormatter.swift]: Using `indentTail()` passed tests! thx suggestion! 8fe61d51d35a459069dfdcf80431ad7a0e331978

File: Sources/Core/Formatter/MultilineFormatter.swift (hunk 2 above)
Comment (sahara-ooga) [reply Sources/Core/Formatter/MultilineFormatter.swift]: That is possible and applied👍🏻 5e572e1be69eef60237f5417ecfeee671a552d66

File: Sources/Core/Formatter/MultilineFormatter.swift [hunk 3]
```diff
@@ -39,10 +39,30 @@ class MultilineFormatter: PrettyFormatter {
         if fields.count == 1, let field = fields.first {
             body = "\(field.0): \(field.1)"
         } else {
-            let contents = fields.map { "\($0): \($1)" }.joined(separator: ",\n")
-            body = contents.indentTail(size: prefix.count)
+            body = fields.map(indentInsertedKeyValueString(_:))
```
Comment (YusukeHosonuma) [inline Sources/Core/Formatter/MultilineFormatter.swift]: May I it make inlining? 😅 

e.g.

```swift
body = fields
    .map { label, value in "\(label): \(value.indentTail(size: "\(label): ".count))" }
    .joined(separator: ",\n")
    .indentTail(size: prefix.count)
```

**Review by YusukeHosonuma — APPROVED** [review]
Good job!

File: Sources/Core/Formatter/MultilineFormatter.swift (hunk 3 above)
Comment (sahara-ooga) [reply Sources/Core/Formatter/MultilineFormatter.swift]: Sure.

File: Sources/Core/Formatter/MultilineFormatter.swift [hunk 4]
```diff
@@ -39,10 +39,30 @@ class MultilineFormatter: PrettyFormatter {
...
+                .joined(separator: ",\n")
+                .indentTail(size: prefix.count)
```
Comment (sahara-ooga) [inline Sources/Core/Formatter/MultilineFormatter.swift]:
```suggestion
            body = fields
                .map(indentInsertedKeyValueString(_:))
                .joined(separator: ",\n")
                .indentTail(size: prefix.count)
```

File: Sources/Core/Formatter/MultilineFormatter.swift (hunk 3 above)
Comment (sahara-ooga) [reply Sources/Core/Formatter/MultilineFormatter.swift]: I make `indentInsertedKeyValueString()` inline. 5a2fa8a

**Comment by sahara-ooga** [comment]
[Tuple対応](https://trello.com/c/9JEA1Axp/1237-tuple%E5%AF%BE%E5%BF%9C)

**Pull request merged**